yarn.lock

```diff
@@ -7793,6 +7793,11 @@ get-stream@^5.0.0, get-stream@^5.1.0:
   dependencies:
     pump "^3.0.0"
 
+get-stream@^6.0.0:
```

Comment:
These shouldn't be related with my PR, but I guess these lockfile updates were somehow missed in previous commits.